locales/en.default.schema.json

```diff
@@ -578,6 +578,55 @@
         "title": {
           "label": "Heading"
         },
+        "title_size": {
+          "label": "Heading size",
+          "options__1":{
+            "label": "Small"
+          },
+          "options__2":{
+            "label": "Medium"
+          },
+          "options__3":{
+            "label": "Large"
+          }
+        },
+        "vertical_offset": {
+          "label": "Heading vertical offset"
+        },
+        "horizontal_offset": {
+          "label": "Heading horizontal offset"
```

Comment:
Is there a reason we start with vertical instead of horizontal? If not I would reverse their order to match theme settings offset existing pattern.  cc. @YoannJailin @katycobb

Reply:
Aligned to list horizontal first, good catch @melissaperreault